release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -172,7 +184,7 @@ The following new features are supported on IBM Z and LinuxONE with {product-tit
 * Three-node cluster support.
```

Comment:
I am no expert but just i got pointed out for the punctuation marks at the end of Z notable enhancements, for the purpose of consistency shouldn't those be removed as i see the punctuation nowhere else (Power too)

Reply:
Looks like it's because that first bullet item is two sentences (and therefore needs punctuation). I think I'm going to leave this as-is of the sake of this PR, but if someone with more knowledge about this content wants to tackle that first bullet to see if all of them can be made consistent, I don't think that's a bad idea.